apps/src/weblab2/layout/VerticalLayout.tsx

```diff
@@ -23,18 +34,16 @@ const VerticalLayout: React.FunctionComponent<LayoutProps> = ({
   isProjectLevel,
   isWidgetView,
 }) => {
-  const widgetViewShowCode = useAppSelector(
-    state => state.codebridgeWorkspace.widgetViewShowCode
-  );
-  const shouldHideEditor = isWidgetView && !widgetViewShowCode;
+  const [viewMode, setViewMode] = useState<ViewMode>(ViewMode.SPLIT);
```

Comment:
This will probably end up in Redux once levelbuilders can control the initialized view mode per this ticket: https://codedotorg.atlassian.net/browse/AFL-119